extensions/url-tools/package.json

```diff
@@ -4,26 +4,46 @@
   "title": "URL Tools",
   "description": "Quickly encode or decode URL from the clipboard",
   "icon": "command-icon.png",
-  "author": "joshua46",
+  "author": "Joshua",
+  "contributors": [
+    "ProAlexUSC"
+  ],
   "license": "MIT",
   "commands": [
     {
       "name": "encode",
-      "title": "URL Encode",
+      "title": "Encode",
+      "description": "Input and get an encoded string, support url encode and base64",
+      "mode": "view"
+    },
+    {
+      "name": "decode",
+      "title": "Decode",
+      "description": "Input and get an decoded string, support url decode and base64",
+      "mode": "view"
+    },
+    {
+      "name": "encodeFromClipboard",
+      "title": "URL Encode from Clipboard",
       "description": "Overwrite the clipboard contents with an URL encoded string",
       "mode": "no-view"
     },
     {
-      "name": "decode",
-      "title": "URL Decode",
+      "name": "decodeFromClipboard",
```

Comment:
Could we keep `decode` here, otherwise we'll lose ranking and if there is a hotkey/alias set